### ISSUE TITLE
dijit/Fieldset.js + css2

### ISSUE THREAD

**Issue by GordonSmith** [OpenNTF/dojo-webpack-plugin#57]
First of all I would like to say a very big thank you for your work in this package - it has added a new lease of life to the whole dojo ecosystem for us!

The issue I am seeing is whenever dijit/Fieldset is referenced within a widget, we get the following webpack issue:
```
ERROR in ./node_modules/dijit/Fieldset.js
Module not found: Error: Can't resolve 'css2' in '/home/gordon/git/HPCC-Platform/esp/src/node_modules/dijit'
 @ ./node_modules/dijit/Fieldset.js 1:0-41:2
 @ ./eclwatch/LZBrowseWidget.js
 @ ./eclwatch/Utility.js
 @ ./eclwatch/stub.js
```
It also looks like there may be several similar occurrences within dijit...

**Comment by GordonSmith** [comment]
I "think" adding:  		
```
new webpack.NormalModuleReplacementPlugin(/^dojo\/query!css2/, "dojo/query"),
```
Fixes but am not 100% if that is the correct approach.

**Comment by chuckdumont** [comment]
That solution will just load the default query selector instead of the css2 query selector.  You should be able to use the loaderProxy here, without specifying any dependencies:

```javascript
new webpack.NormalModuleReplacementPlugin(
	/^dojo\/query!/, function(data) {
		data.request = "dojo/loaderProxy?loader=" + data.request;
	}
)
```

**Comment by GordonSmith** [comment]
Great thanks!

**Comment by GordonSmith** [comment]
@chuckdumont Question:  Should these "recipes" end up in the dojo-webpack-plugin as defaults?

**Comment by chuckdumont** [comment]
I was just thinking I should add this one.  Thanks for pointing it out.

**Comment by GordonSmith** [comment]
No problem.  FWIW I am now successfully packaging a large web application successfully.  It really has opened the door to being able extend and modernize it with new functionality without having the headache of coercing pieces into the build process - so many thanks again to yourself...

**Comment by GordonSmith** [comment]
@chuckdumont BTW the above gives the following error:
```
ERROR in ./node_modules/dijit/form/_RadioButtonMixin.js
Module not found: Error: Can't resolve 'css2' in '/home/gordon/git/HPCC-Platform/esp/src/node_modules/dijit/form'
```

**Comment by chuckdumont** [comment]
So it turns out that the loaderProxy needs an enhancement to deal with the case where the resource name (e.g. `css2` in this case) does not represent a module.   Webpack's loader syntax does not allow non-module resource names in loader expressions, so I've added the *name* query arg to the loaderProxy URL syntax to accommodate these types of cases.  See #58.

**Comment by chuckdumont** [comment]
The fix is available in release 2.1.1
